modules/src/ics24_host/identifier.rs

```diff
@@ -101,6 +101,12 @@ impl FromStr for ClientId {
     }
 }
 
+impl Default for ClientId {
```

Comment:
This is required by abscissa for `Options` and because the positional fields not being `Option` i think. Didn't look much into it to be honest, just reacted to the compiler complaint.